types/milesight/VS133/region_count.schema.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$id": "https://akenza.io/xovis/VS133/region_count.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "processingType": "uplink_decoder",
+  "topic": "region_count",
+  "title": "Region count",
+  "properties": {
+    "region1count": {
+      "title": "Region 1 count",
+      "description": "Region 1 count",
+      "type": "integer"
```

Comment:
what is the unit here, people?

Reply:
Yes, but we usually do not give a unit for those. I think thats because of the changing plural and singular, depending on the count.